vendors/pmx/models/pmx-tcr/model.yaml

```diff
@@ -0,0 +1,58 @@
+# Commercial name of the model
+name: PMX TCR
+# Functional description of the product. Maximum 500 characters.
+description: "Radar Traffic Counter" 
+# Logo of the device
+logo: pmx-tcr.png
+# ID(s) of the profile that defines the LoRaWAN characteristics of this model.
+# id: as defined under /profiles.
+# lorawanDeviceProfileID: as defined by LoRa Alliance in https://lora-alliance.org/wp-content/uploads/2020/10/LoRa_Alliance_Vendor_ID_for_QR_Code_02142022.pdf.
+# The lorawanDeviceProfileID consists of 8 upper-case hexadecimal characters, 4 characters for VendorID + 4 characters for VendorProfileID. Please consider using single (') or double (") quotation.
+deviceProfileIds:
+  - id: pmx_RFGroup1_1.0.4_classA
+  - id: pmx_RFGroup1_1.1_classA
+  - id: pmx_RFGroup3_1.0.4_classA
+  - id: pmx_RFGroup3_1.1_classA
+  - id: pmx_RFGroup4_1.0.4_classA
+  - id: pmx_RFGroup4_1.1_classA
+
+# You may optionally customize any of the following settings to override the generic value set in LoRaWAN device profiles associated with your model. Leave empty if you want to keep the Device Profile settings.
+# Maximum device TX Conducted output power in dBm.
+maxTxPower:
+# Minimum device TX Conducted output power in dBm.
+minTxPower:
+# Maximum device TX Radiated output power in dBm.
+maxTxEIRP:
+# Minimum device TX Radiated output power in dBm.
+minTxEIRP:
+# Typical mobility profile of the device. Possible values are 'near_static' (also valid for static devices), 'walking_speed', 'vehicular_speed'
+# or 'random' (not known, changes over time).
+motionIndicator:
+
+# Is your device certified by the LoRa Alliance? Possible values: true, false.
+LoRaWANCertified: false
+# Always mandatory: `<vendorId>:<modelName>:<modelVersion>` (You should have only one per model)
+# Format: `<8 characters max>:<16 characters max>:<digits only>`
+# Be careful: Once you chose a model, it cannot be changed later
+modelId: pmx:pmx-tcr:5
+# <vendorId>:<protocolName>:<protocolVersion> Needed for linking the model with a specific driver -> must be the same one used in driver.yaml in the corresponding model (You might have several ones)
+# Be careful: Once you chose a protocolId, it cannot be changed later
+protocolId:
+  - pmx:pmx-tcr-protocol:1
+# DataSheet URL (optional)
+specificationURL:
+# User Guide URL (optional)
+userGuideURL:
+# <sensor>:<unitId> Available sensors following Actility ontology: https://github.com/actility/device-catalog/blob/main/template/sample-vendor/drivers/ONTOLOGY.md
+sensors:
+
+# Driver examples
+# A list of examples description that are compatible with this model
+examples:
+  pmx:pmx-tcr-protocol:1:
```

Comment:
I see that you have added a list of examples descriptions related to the driver with protocolId "pmx:pmx-tcr-protocol:1" (which does not exist). Is the driver missing by mistake? or this list should be removed?